components/Buttons/src/MDCFloatingButton+Animation.m

```diff
@@ -196,7 +197,7 @@ - (void)collapse:(BOOL)animated completion:(void (^_Nullable)(void))completion {
                                                             self.imageView.layer.transform,
                                                             [MDCFloatingButton collapseTransform])]
                    fromValue:nil
-              timingFunction:[[CAMediaTimingFunction alloc] initWithControlPoints:0.4f:0:1:1]
+              timingFunction:[[CAMediaTimingFunction alloc] initWithControlPoints:(float) 0.4:0:1:1]
```

Comment:
nite: remove extra space

Reply:
I don't really want to fight clang-format on these right now. It's what happens when you format the code.